developer/python-django-devtools-reference.rst

```diff
@@ -0,0 +1,244 @@
+Python/django dev tools reference
+=================================
+
+In this page, we aim at helping technical users and contributors to work
+on OpenWISP and we introduce to some python tools they can find very
+useful during development and debugging.
+
+Python and its advantages
+-------------------------
+
+As you may already know, `python <https://www.python.org>`_ is a
+programming language, more specifically an interpreted high-level
+programming language for general-purpose programming. It is also widely
+used these days! Big companies like Google, Disney and Dropbox make
+excessive use of this programming language! You'll know why in an instant.
```

Comment:
I would write `You'll know why in a moment.`. I didn't know that use of `instant`. To me `moment` is much more common. Of course this is nitpicking :-)